judge/admin/problem.py

```diff
@@ -248,3 +249,23 @@ def construct_change_message(self, request, form, *args, **kwargs):
         if form.cleaned_data.get('change_message'):
             return form.cleaned_data['change_message']
         return super(ProblemAdmin, self).construct_change_message(request, form, *args, **kwargs)
+
+
+class ProblemPointsVoteAdmin(admin.ModelAdmin):
+    list_display = ('points', 'voter', 'problem', 'vote_time')
```

Comment:
Make `problem` link to the problem. See `CommentAdmin.linked_page` for a code example.

Reply:
done